### ISSUE TITLE
10.7.0 backport was not published

### ISSUE THREAD

**Issue by xvik** [rnveach/checkstyle-backport-jre8#9]
Backport for 10.7.0 stated as released but it was not published (404):

https://rnveach.github.io/checkstyle-backport-jre8/maven2/com/puppycrawl/tools/checkstyle-backport-jre8/10.7.0/checkstyle-backport-jre8-10.7.0.pom  

Previous version 10.6.0 is available (so url above should be correct):
https://rnveach.github.io/checkstyle-backport-jre8/maven2/com/puppycrawl/tools/checkstyle-backport-jre8/10.6.0/checkstyle-backport-jre8-10.6.0.pom

**Comment by xvik** [comment]
Thank you for explanation!
I see 10.7.1 should be released in a week, untill then I'll stay on 10.6.

**Comment by rnveach** [comment]
If next release burps again, I will see what I can do for fixing this and not wait for the one after. I am not sure what I can do if it does happen since there is no manual retry and no description of the error.

**Comment by rnveach** [comment]
Backport was successfully published to GH pages.
https://rnveach.github.io/checkstyle-backport-jre8/maven2/com/puppycrawl/tools/checkstyle-backport-jre8/maven-metadata.xml

I was able to watch the status in real time at https://github.com/rnveach/checkstyle-backport-jre8/actions/runs/4272927420 . I disabled the actions before. I wonder if something changed with GH that it does the deployment through actions.
